### PR TITLE
fix: correct date picker

### LINKED ISSUE #49: fix/ui-date-picker-timezone
when registering and entering date 07/17/2005 it is sent with +0 tz
while api saves dates to database with its local timezone meaning +2 tz
it ends up being saved as 07/16/2005 in database
we need to correct that, ignore timezone from client or dont save it in api

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

File: src/screens/Register/Register.tsx [hunk 1]
```diff
@@ -86,9 +86,14 @@ export const Register = () => {
                 </Typography>
 
                 <form
-                    onSubmit={handleSubmit((data) =>
-                        registerHook
-                            .mutateAsync(data)
+                    onSubmit={handleSubmit((data) => {
+                        const birthdate = new Date(data.birthdate);
+                        birthdate.setDate(birthdate.getDate() + 1);
```
Comment (maciejopalinski) [inline src/screens/Register/Register.tsx:91]: Very sus
What is your solution? How it works?

I think that we might just change the birthdate column type in API so it doesn't store the timezone info.

**Review by maciejopalinski — CHANGES_REQUESTED** [review]

File: src/screens/Register/Register.tsx (hunk 1 above)
Comment (T0ffik) [reply src/screens/Register/Register.tsx:91]: I know it's not best solution but works, if date in database is always one day before, it's giving next day that user passed in register form

Comment (T0ffik) [reply src/screens/Register/Register.tsx:91]: We can just leave this solution for now, if we don't get better merge this before demo

Comment (maciejopalinski) [reply src/screens/Register/Register.tsx:91]: Ok let's leave it for now. If i don't come up with a better solution we will merge it just yours before demo

**Review by Mariusz-web — APPROVED** [review]

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**